src/processTargets.ts

```diff
@@ -443,8 +479,12 @@ function getLineSelectionContext(
   selection: SelectionWithEditor,
   selectionContext: SelectionContext
 ): SelectionContext {
-  const document = selection.editor.document;
-  const { start, end } = selection.selection;
+  if (selectionContext.isInDelimitedList) {
+    return selectionContext;
+  }
```

Comment:
We actually do want to clobber the delimiter for "line", because they've explicitly asked for it

Reply:
Yes we should probably remove that if statement